cherrypy/lib/static.py

```diff
@@ -94,6 +94,7 @@ def serve_file(path, content_type=None, disposition=None, name=None,
     #   this way CP won't load the whole file in memory
     content_length = st.st_size
     fileobj = open(path, 'rb')
+    cherrypy.serving.request.path_abs = path
```

Comment:
Simply the fact that I didn't find the absolute (local)  path anywhere. It's quite likely that I missed out somewhere...